Misc/NEWS.d/3.14.0a1.rst

```diff
@@ -2119,9 +2119,8 @@ imported.  Patch by Sergey B Kirpichev.
 .. nonce: W9Gd09
 .. section: Library
 
-Support the :c:expr:`float complex` and :c:expr:`double complex` C types in
-the :mod:`struct` module if the compiler has C11 complex arithmetic.  Patch
-by Sergey B Kirpichev.
+Support the :c:expr:`float complex` and :c:expr:`double complex` C types in the
```

Comment:
Considering this is in 3.14.0a1.rst, I would suggest removing it from 3.14.0a1.rst and move it to 3.14.0a7.rst. In 3.14.0a1.rst, the tarball only contained something that was available for a compiler supporting C11 complex arithmetic.

Reply:
> Considering this is in 3.14.0a1.rst, I would suggest removing it from 3.14.0a1.rst and move it to 3.14.0a7.rst. In 3.14.0a1.rst, the tarball only contained something that was available for a compiler supporting C11 complex arithmetic.

I'm not sure what we gain here, could you please explain?  Both a1 and a7 have support, which is restricted.

Reply:
Wait I may be missing something. Now we have unconditional support right? before if was guarded by a conditional #ifdef so it's a bit different right? What I want to say is that, the NEWS entry should be as close as the reality, namely what's in the a1 release tarball should be mentioned and vice-versa. I read this NEWS entry as "we unconditionally support float complex and double complex C types", which is only the case for a7.

But it's maybe too nitpicky (I don't know how precise the RM (@hugovk) wants the changelog to be)

Reply:
> Now we have unconditional support right?

Yes.  But not in any alpha yet.

Reply:
Ah yes, I incorrectly said a7. I wanted to say in "beta". My bad. Since the tarball is tagged, we can simply do the following:

- remove the blurb entry in a1.rst
- add a blurb entry using blurb so that it's in next and appears in the latest page

That way, users downloading the a1 tarball will have the old news entry while anyone downloading something after the beta will have the new NEWS entry.